Dragonflight/WarriorArms.lua

```diff
@@ -381,8 +390,8 @@ spec:RegisterAuras( {
     },
     test_of_might = {
         id = 385013,
-        duration = 3600,
-        max_stack = 1
+        duration = 12,
+        max_stack = 1, -- TODO: Possibly implement fake stacks to track the Strength % increase gained from the buff
```

Comment:
Can you test to see if the actual aura includes information about the strength % increase?  If so, we can add a meta function to replace the actual stack value.

Reply:
I know from testing this and fooling around with it in-game that the only thing that happens is the buff's %strength increase in the tooltip is updated to whatever Test of Might resulted to in the buildup. There is only 1 (one) stack of this buff though for any% increase.

Reply:
This might not be worth digging into, because the other question is "would this information ever be used to make a combat decision?"  If there's never going to be a priority list written that accounts for stacks/values, we don't really need to know.

Reply:
At this point in time I don't think it's worth digging into until after APLs are established.